projects/plugins/jetpack/extensions/shared/external-media/sources/index.js

```diff
@@ -25,6 +31,12 @@ export const mediaSources = [
 		icon: <PexelsIcon className="components-menu-items__item-icon" />,
 		keyword: 'pexels',
 	},
+	{
+		id: SOURCE_OPENVERSE,
+		label: __( 'Openverse', 'jetpack' ),
+		icon: <OpenverseIcon className="components-menu-items__item-icon" />,
+		keyword: 'openverse',
+	},
```

Comment:
Can y'all hide this behind a feature flag of some sort? I'm not familiar with Jetpack enough to know if that's possible, but in case Openverse's API can't handle the new load and needs to be scaled, it would be nice to be able to prevent requests from coming in while the scaling happens.

Reply:
I'm not aware of feature flags in Jetpack. Once something is enabled in the plugin it would require a plugin update on self-hosted sites to change features like that. 
Having said that, we have a bit more control about simple and atomic sites and could roll it back there if needed.